plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -143,3 +145,45 @@ def package_signature(headers):
         return None
     # gpg program uses the last 8 characters of the fingerprint
     return signature.split()[-1][-8:]
+
+
+def signature_enabled(config):
+    """
+    Check if the signature policy is enabled.
+
+    :param config: configuration instance passed to the importer
+    :type  config: pulp.plugins.config.PluginCallConfiguration
+
+    :return: true if enabled, false otherwise
+    :rtype: boolean
+    """
+
+    require_signature = config.get("require_signature", False)
```

Comment:
Both of these setting names would be good to have in a ``constants`` module for DRY reasons.